README.md

```diff
@@ -5,7 +5,7 @@ Feedstock license: [BSD-3-Clause](https://github.com/conda-forge/itslive-feedsto
 
 Home: https://github.com/nasa-jpl/itslive-vortex
 
-Package license: GPL-3.0
+Package license: BSD-3-Clause
```

Comment:
Package license is still GPL-3.0 no? https://github.com/nasa-jpl/itslive-py/blob/main/LICENSE

Reply:
Oops, @betolink, could you open another PR to patch the license, and maybe update the URLs from https://github.com/nasa-jpl/itslive-vortex to https://github.com/nasa-jpl/itslive-py

Reply:
Yeah I updated that just before merging, got confused with the license from the feedstock, yes the license is still GPLv3. Thanks for noticing Wei Ji! also would you want to be on the maintainers list? (both the library and the conda-forge repos)

Reply:
Maybe just the conda-forge repo, I'm less familiar with what the Python client library is doing :slightly_smiling_face: